src/execution_tree/primitives/add_operation.cpp

```diff
@@ -30,6 +30,57 @@ HPX_REGISTER_DERIVED_COMPONENT_FACTORY(
 HPX_DEFINE_GET_COMPONENT_TYPE(add_operation_type::wrapped_type)
 
 ///////////////////////////////////////////////////////////////////////////////
+
+
+struct AddScalar
```

Comment:
We have specific naming conventions, please see here for a short description: https://github.com/STEllAR-GROUP/hpx/wiki/HPX-Source-Code-Structure-and-Coding-Standards. All variable, function, and type names are to be lower-case only, possibly words separated by underscores (i.e. `add_scalar` instead of `AddScalar`). Template parameters should be `CamelCase` and macros all `UPPER_CASE`.